src/ray/protobuf/event.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package ray.rpc;
+
+message Event {
+  enum SourceType {
+    COMMON = 0;
```

Comment:
What is the common source type? Can you write a comment here?

Reply:
The COMMON type is designed for some process that does not init there own source_type.
We suppose that there will be some roles in addition to GCS, RAYLET, CORE_WORKER. There is no clear application for this type now.

Reply:
It'll be great if we can write what you just said as a comment!